docs/sources/configure/metrics-traces-attributes.md

```diff
@@ -279,3 +279,23 @@ Beyla periodically resynchronizes the whole Kubernetes metadata at the frequency
 by this property.
 
 Higher values reduce the load on the Kubernetes API service.
+
+## Extra group attributes
+
+Beyla supports adding custom attributes to metrics by configuring `extra_group_attributes`. You need to add the desired group and an array of attributes to be added to this group.
+
```

Comment:
I believe we should add here a table like the ones above to define the yaml field

Reply:
Added a table describing group labels and metrics with related groups